fuzz/CMakeLists.txt

```diff
@@ -21,6 +21,8 @@ fuzzer(der_roundtrip)
 fuzzer(dtls_client ssl)
 fuzzer(dtls_server ssl)
 fuzzer(ocsp)
+fuzzer(ocsp_http)
```

Comment:
Where is this fuzzer defined? I don't get why what this is building and why it's not failing.

Reply:
Looks like it is already there in the tree, just wasn't enabled.